docs/physics/index.rst

```diff
@@ -2,5 +2,66 @@
 
 .. _plasmapy-physics:
 
+****************************************
+The Physics Package (`plasmapy.physics`)
+****************************************
+
+.. currentmodule:: plasmapy.physics
+
+Introduction
+============
+
+`plasmapy.physics` provides theoretical formulas for calculation of physical quantities helpful for plasma physics.
+The layout of the subpackage is still in flux, but for now we have settled on providing a single `plasmapy.physics` namespace
+for some of the most common functions. The actual functions are located in modules, subjectively grouped by topic.
```

Comment:
With sub-namespaces for larger modules like transport?

Reply:
Sure. I'll add that after work.